app/features/officer/forms/OfficerForms.scala

```diff
@@ -124,15 +124,29 @@ object ContactDetailsForm {
   val form = Form(
     mapping(
       EMAIL         -> optional(text.verifying(StopOnFirstFail(FormValidation.IsEmail(s"$FORM_NAME.$EMAIL"),maxLenText(EMAIL_MAX_LENGTH)))),
-      DAYTIME_PHONE -> optional(text.transform(removeSpaces, identity[String]).verifying(regexPattern(PHONE_NUMBER_PATTERN)(s"$FORM_NAME.$DAYTIME_PHONE"))),
-      MOBILE        -> optional(text.transform(removeSpaces, identity[String]).verifying(regexPattern(PHONE_NUMBER_PATTERN)(s"$FORM_NAME.$MOBILE")))
+      DAYTIME_PHONE -> optional(text.transform(removeSpaces, identity[String]).verifying(isValidPhoneNumber)),
```

Comment:
couldnt these just be DAYTIME_PHONE -> option(text).verifying(isValidPhoneNumber("FormNameHere")